src/components/Positions/Positions.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import Button from '../Button';
+
+export interface PositionsProps {}
+
+const data = {
+  title: 'Výpis aktuálních pozic',
+
+  items: [
+    {
+      position: 'Dermatolog',
+      img: '/assets/medicon/images/positions-1.jpg'
+    },
+    {
+      position: 'Gynekolog',
+      img: '/assets/medicon/images/positions-2.jpg',
+    },
+    {
+      position: 'Ortoped',
+      img: '/assets/medicon/images/positions-3.jpg',
+    },
+  ]
+};
+
+const Positions = (props: PositionsProps) => {
```

Comment:
position is very general name, ad something a bit more escriptive, maybe JobPositions or career Positions? :)